hwy/ops/x86_256-inl.h

```diff
@@ -275,6 +275,42 @@ HWY_API Vec256<double> Undefined(D /* tag */) {
 
 HWY_DIAGNOSTICS(pop)
 
+// ------------------------------ ResizeBitCast
+
+// 32-byte vector to 32-byte vector (or 64-byte vector to 64-byte vector on
+// AVX3)
+template <class D, class FromV, HWY_IF_V_SIZE_GT_V(FromV, 16),
+          HWY_IF_V_SIZE_D(D, HWY_MAX_LANES_V(FromV) * sizeof(TFromV<FromV>))>
+HWY_API VFromD<D> ResizeBitCast(D d, FromV v) {
+  return BitCast(d, v);
+}
+
+// 32-byte vector to 16-byte vector (or 64-byte vector to 32-byte vector on
+// AVX3)
+template <class D, class FromV, HWY_IF_V_SIZE_GT_V(FromV, 16),
+          HWY_IF_V_SIZE_D(D,
+                          (HWY_MAX_LANES_V(FromV) * sizeof(TFromV<FromV>)) / 2)>
```

Comment:
Here and above we could also use DFrom<FromV>().MaxBytes() (/2)?

Reply:
I was actually getting compilation errors on MSVC when I tried to replace `(HWY_MAX_LANES_V(FromV) * sizeof(TFromV<FromV>)) / 2` with `DFromV<FromV>().MaxBytes() / 2`.

Need to use the `(HWY_MAX_LANES_V(FromV) * sizeof(TFromV<FromV>)) / 2` here as x86_256-inl.h needs to be compilable with MSVC.

Reply:
I see, thanks for trying it :/ MSVC does indeed struggle with SFINAE.